src/commons/api/history.rs

```diff
@@ -537,6 +538,11 @@ impl WithStorableDetails for StorableCaCommand {
             } => CommandSummary::new("cmd-ca-rcn-receive", &self)
                 .with_rcn(resource_class_name)
                 .with_resources(resources),
+            StorableCaCommand::DropResourceClass {
+                resource_class_name,
+                ..
```

Comment:
Is ignoring the drop reason here deliberate?

Reply:
Yes, well kind of, the reason is a string rather than a type and cannot be translated. We include a map of key-values to help translations. That said.. losing it altogether is not the best choice. So including it now.